projects/novo-elements/src/elements/common/overlay/Overlay.ts

```diff
@@ -219,7 +221,10 @@ export class NovoOverlayTemplateComponent implements OnDestroy {
   protected createOverlay(template: TemplateRef<any>): void {
     this.portal = new TemplatePortal(template, this.viewContainerRef);
     this.overlayRef = this.overlay.create(this.getOverlayConfig());
-    this.overlayRef.backdropClick().subscribe(() => this.closePanel());
+    this.overlayRef.backdropClick().subscribe(() => {
+      this.backDropClicked.emit(true);
```

Comment:
I may be unaware of something else, but is there something we need this for that doesn't work with just the `closing` event, signifying the overlay closing? (It could also be important if someone closes the overlay another way like pressing the Esc key)

Reply:
Yes, a backdrop click in `novo > advanced-search.component` is now effectively a CANCEL close, as opposed to a SUBMIT close. IOW, all cancels are closing events, but not all closing events are cancels.

If Product requests the ESC key be included in the future, we can add that later, but this optional event emission should suffice for now, I think.